assets/js/script.js

```diff
@@ -95,25 +95,43 @@ $(document).on('click', '.js-bag-link', function(evt) {
 $(document).on('click', '.js-remove-letter', function(evt) {
   var letters = state.letters.split('')
   letters.splice(state.activeLetter, 1)
-
   setState({
     letters: letters.join(''),
     editing: false,
   })
 })
 
-
 // edit individual letter
 $(document).on('click', '.js-letter', function(evt) {
   evt.stopPropagation()
 
   setState({
-    editing:      true,
-    editingBag:   false,
+    editing: true,
+    editingBag: false,
     activeLetter: $(this).data('index'),
   })
 })
 
+$(document).on('click', '.js-tab', function(evt) {
+  evt.stopPropagation()
```

Comment:
not sure you need this `stopPropagation`. usually you only need it if there's another element in the DOM hierarchy that the click would bubble up to and you don't want to trigger that behavior.